dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/consent/ConsentDetails.java

```diff
@@ -0,0 +1,96 @@
+package edu.harvard.iq.dataverse.persistence.consent;
+
+import edu.harvard.iq.dataverse.persistence.config.LocaleConverter;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import java.util.Locale;
+
+@Entity
+public class ConsentDetails {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(nullable = false, name = "consent_id")
+    private Consent parent;
+
+    @Column(nullable = false, updatable = false)
+    @Convert(converter = LocaleConverter.class)
+    private Locale textLanguage;
```

Comment:
I'd go for `language` instead of `textlanguage`. The way it's modeled right now, `Locale` field describes `ConsentDetails` and not just `ConsentDetails.text` (although in reality it does just that)